[TASK] Describe the core link handler (#2411)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/LinkBrowser/CoreLinkHandler.rst

.. include:: /Includes.rst.txt
.. index:: Core link handler
.. _core-link-handler:

=================
Core link handler
=================

> The core link handler, implementing the

Reviewer: ```suggestion
The Core link handler, implementing the
```

> :php:`TYPO3\CMS\Core\LinkHandling\LinkHandlingInterface` described here

Reviewer: ```suggestion
:php:`\TYPO3\CMS\Core\LinkHandling\LinkHandlingInterface` described here
```

matches between different internal representations of a link. It is not

> to be confused with the :ref:`backend link handler <linkhandler>`, commonly

Reviewer: ```suggestion
to be mixed up with the :ref:`backend link handler <linkhandler>`, commonly
```

maybe better than "confused"

also just called link handler. The latter implements the

> :php:`TYPO3\CMS\Backend\LinkHandler\LinkHandlerInterface` and renders a

Reviewer: ```suggestion
:php:`\TYPO3\CMS\Backend\LinkHandler\LinkHandlerInterface` and renders a
```

tab in the link browser.

> You can find the build-in core link handlers in

Reviewer: ```suggestion
You can find the built-in Core link handlers in
```

:file:`EXT:core/Classes/LinkHandling`.

In the tutorial section we provide an example on how to implement a

> :ref:`custom core link handler <tutorial-core-link-handler>`.

Reviewer: ```suggestion
:ref:`custom Core link handler <tutorial-core-link-handler>`.
```
